src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -375,12 +375,12 @@
   (views/letsubs [identity      [:contacts/current-contact-identity]
                   maybe-contact [:contacts/current-contact]]
     (let [contact (or maybe-contact (contact.db/public-key->new-contact identity))
-          {:keys [pending? public-key]} contact]
+          {:keys [added? public-key]} contact]
       [react/view {:style styles/chat-profile-body}
        [profile.views/profile-badge contact]
        ;; for private chat, public key will be chat-id
        [react/view
-        (if (or (nil? pending?) pending?)
+        (if added?
```

Comment:
maybe I got it the other way around, but should this not be the opposite?
if `added?` we don't want to show the `add-to-contacts` bar?

Reply:
indeed, I tested on android so I missed this as it is desktop